packages/paste-theme/__tests__/themeProvider.spec.tsx

```diff
@@ -20,17 +20,6 @@ describe('Theme.Provider', () => {
     ReactDOM.render(<Theme.Provider />, document.createElement('div'));
   });
 
-  it('should render default link text color instead of console', () => {
-    const {getByText} = render(
-      <Theme.Provider theme="console">
-        <ThemeConsumerExampleTextColor />
-      </Theme.Provider>
-    );
-
-    // console theme link color: rgb(4, 60, 181)
-    expect(getByText('Color: rgb(2, 99, 224)')).toBeDefined();
-  });
-
```

Comment:
Deprecated theme in the theme provider. We don't even import it so there's no point in testing it if it's just the same as default theme.